concordium-std/src/traits.rs

```diff
@@ -133,6 +133,9 @@ where
     /// `.seek(SeekFrom::Start(0))` but can be implemented more efficiently.
     fn move_to_start(&mut self);
 
+    /// Get the cursor position counted from the beginning of the stream.
+    fn get_cursor_position(&self) -> u32;
```

Comment:
Just want to mention that Seek already exposes a way to get the current position. This function is of course more direct.

Reply:
I would probably prefer if we add a method `get_cursor_position` to the Seek trait (with a default implementation).

Reply:
I couldn't add a default implementation because it would need a `&mut self`. But I added it to the `Seek` trait nonetheless.